src/components/ui/Button.jsx

```diff
@@ -0,0 +1,13 @@
+const Button = ({element}) => {
+  const Message = (event) =>{
```

Comment:
Could you start the function with a lower letter, as it is not a component or a class?
```suggestion
  const message = (event) =>{
```